docs/content/about/tech-specs.md

```diff
@@ -16,63 +16,32 @@ The following tables list the software versions NGINX Service Mesh supports and
-|--------------------|-------------|-----------------------------------------|----------------------------------------|------------|----------|----------|
-| v1.7.0             | 1.22 - 1.24 | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 4.9 - 4.10 | 3.2+     | 2.6, 2.7 |
-| v1.6.0             | 1.22 - 1.24 | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 4.9 - 4.10 | 3.2+     | 2.5, 2.6 |
-| v1.5.0             | 1.22 - 1.24 | 1.12, 2.1, 2.2                          | 2.2                                    | 4.9 - 4.10 | 3.2+     | 2.5, 2.6 |
-| v1.4.1             | 1.18 - 1.22 | 1.12, 2.1, 2.2                          | 2.2                                    | 4.8        | 3.2+     | 2.5, 2.6 |
-| v1.4.0             | 1.18 - 1.22 | 1.12, 2.0, 2.1                          | 1.12, 2.0, 2.1                         | 4.8        | 3.2+     | 2.5, 2.6 |  
-| v1.3.1             | 1.18 - 1.22 | 1.12, 2.0                               | 1.12, 2.0                              | 4.8        | 3.2+     | 2.5, 2.6 |     
-| v1.3.0             | 1.18 - 1.22 | 1.12, 2.0                               | 1.12, 2.0                              | 4.8        | 3.2+     | 2.5, 2.6 |     
-| v1.2.1             | 1.16 - 1.21 | 1.12.0                                  | 1.12.0                                 | 4.8        | 3.5, 3.6 | 2.6      |
-| v1.2.0             | 1.16 - 1.21 | 1.12.0                                  | 1.12.0                                 | 4.8        | 3.5, 3.6 | 2.6      |
-| v1.1.0             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-| v1.0.1             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-| v1.0.0             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-
+| NGINX Service Mesh | Kubernetes\*  | OpenShift\*   | NGINX Plus Ingress Controller (ingress) | NGINX Plus Ingress Controller (egress) | Helm     | Rancher  |
+|--------------------|---------------|---------------|-----------------------------------------|----------------------------------------|----------|----------|
+| v1.7.0             | 1.24 - 1.26   | 4.10 - 4.11   | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 3.2+     | 2.6, 2.7 |
+| v1.6.0             | 1.22 - 1.24   | 4.9 - 4.10    | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 3.2+     | 2.5, 2.6 |
 {{% /table %}}
 
+\* Versions listed have been tested and verfied to work with NGINX Service Mesh. Other versions may work, but given the scope of available versions and platforms,
```

Comment:
verified*

Reply:
we are making an excuse.  This reflects poorly.
If you want a disclaimer note:
Currently released versions of Kubernetes are validated.  When known incompatibilities are identified that limit the versions of Kubernetes that could possibly be supported by a release of NGINX Service Mesh it will be noted.

An absolute 'support' statement is limiting, even versions tested in a table.
When we have a known breaking version, we should note it.

If Service Mesh builds on some new feature of K8s, then we will have a break and a version limitation.  Otherwise, stating a version support matrix is artificial.  Because old versions of K8s never change.  And unless Service Mesh is using a new interface or we know some dependent feature of K8s has changed, then the supported version should not change.

This all relates to K8s Platform Conformance.
And not defining a narrow support range artificially improves the reach / attractiveness of the product.
Right now, the fact that NSM is not tightly bound to specific K8s versions makes it more attractive than Istio.

Reply:
Removed this disclaimer.